src/vivarium/framework/lookup/table.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         self,
         table_number: int,
         data: pd.DataFrame,
-        population_view_builder: Callable[[list[str]], PopulationView],
+        population_view_builder: Callable[[LookupTable, list[str]], PopulationView],
```

Comment:
The `population_view_builder` is `builder.population.get_view` which doesn't take a `LookupTable` as its first argument (it usually takes a `Component`. This component is only passed to `get_view` so that the `PopulationView` knows which private columns it is allowed to update. Lookup tables are never allowed to update private columns - they are exclusively read-only - so this argument is essentially irrelevant. I think it makes more sense to pass `None` here to signify that it has no access to any private columns.